ci/dev/test-e2e.sh

```diff
@@ -15,7 +15,7 @@ main() {
     echo -e "\n"
     exit 1
   fi
-  CS_DISABLE_PLUGINS=true ./test/node_modules/.bin/jest "$@" --config ./test/jest.e2e.config.ts
+  CS_DISABLE_PLUGINS=true ./test/node_modules/.bin/jest "$@" --config ./test/jest.e2e.config.ts --runInBand
```

Comment:
> Run all tests serially in the current process, rather than creating a worker pool of child processes that run tests

[Jest Docs](https://jestjs.io/docs/cli#--runinband)

This reduces flakiness of tests.

Reply:
Surprised that running it out of band is the default! Multiple instances of `code-server` logging in and out at the same time doesn't sound fun in retrospect.